benches/mantis/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
I think we can get rid of that because it's exactly the same license as the current repo.